### PR TITLE
Update ingress-controllers.md

### PR DESCRIPTION
**Pull request by liuqi-sun**
Added OpenNJet Ingress Controller

### CODE REVIEW

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
   works with the [NGINX](https://www.nginx.com/resources/glossary/nginx/) webserver (as a proxy).
 * The [ngrok Kubernetes Ingress Controller](https://github.com/ngrok/kubernetes-ingress-controller) is an open source controller for adding secure public access to your K8s services using the [ngrok platform](https://ngrok.com).
 * The [OCI Native Ingress Controller](https://github.com/oracle/oci-native-ingress-controller#readme) is an Ingress controller for Oracle Cloud Infrastructure which allows you to manage the [OCI Load Balancer](https://docs.oracle.com/en-us/iaas/Content/Balance/home.htm).
+* [OpenNJet Ingress Controller](https://gitee.com/njet-rd/open-njet-kic) is a [OpenNJet](https://njet.org.cn/)-based ingress controller.
```
Comment (windsonsea) [inline content/en/docs/concepts/services-networking/ingress-controllers.md:62]: It's not suitable to add a project hosted on [gitee.com](https://gitee.com)

**Review by windsonsea — COMMENTED** [review]
Please sign CLA: 
https://github.com/kubernetes/community/blob/master/CLA.md

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:62]: I don't know of any reason for concern _by Kubernetes_. Although there have been complaints of censorship, we allow links to closed-source software as well as truly open source projects.

**Review by Arhell — COMMENTED** [review]
/easycla

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (liuqi-sun) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:62]: Can I link to gitee?

Comment (sftim) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:62]: You can.

Comment (liuqi-sun) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:62]: > You can.

What else is wrong with my PR?Is it acceptable?

**Review by sftim — COMMENTED** [review]
/lgtm
This looks right to me.

**Review by divya-mohan0209 — COMMENTED** [review]
Since there is a thirdparty content shortcode already in place, there doesn't seem to be a reason why with a valid link, inclusion of the OpenNJet Ingress Controller isn't okay. 
/remove-lifecycle stale
/approve

**Pull request merged**